src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/StandardOleMarshalObject.Windows.cs

```diff
@@ -118,4 +118,23 @@ int IMarshal.DisconnectObject(int dwReserved)
             return HResults.E_NOTIMPL;
         }
     }
+
+    [ComImport]
```

Comment:
We tend to prefer one type per file, but I think it is ok to put it together in this case.

Reply:
Agreed. I was torn between both doing that and adding it to the usage place. I chose this approach because I don't want `IMarshal` to creep into other places and so keeping this tightly coupled implies that.